codecov.yml

```diff
@@ -2,6 +2,6 @@ coverage:
   status:
     project:
       default:
-        target: 80%
```

Comment:
Not achievable right now, we're at 60% with the unit test tickets. This can be bumped up in the future